### ISSUE TITLE
Maximum call stack size exceeded

### ISSUE THREAD

**Issue by tourman** [telegraf/telegraf#1110]
## Context

Unable to understand the reason of the thrown exception.

* Telegraf.js Version: 3.32.0
* Node.js Version: 13.8.0
* Operating System: CentOS 7

## Expected Behavior

No unexpected exception or a proper call stack at least with ability to understand the real reason.

## Current Behavior

```
Exception in PromiseRejectCallback:
/opt/project/node_modules/telegraf/composer.js:355
          return Promise.reject(err)
                         ^

RangeError: Maximum call stack size exceeded

  RangeError: Maximum call stack size exceeded
      at execute (/opt/project/node_modules/telegraf/composer.js:338:24)
      at /opt/project/node_modules/telegraf/composer.js:337:14
      at execute (/opt/project/node_modules/telegraf/composer.js:352:13)
      at /opt/project/node_modules/telegraf/composer.js:337:14
      at execute (/opt/project/node_modules/telegraf/composer.js:352:13)
      at /opt/project/node_modules/telegraf/composer.js:337:14
      at execute (/opt/project/node_modules/telegraf/composer.js:352:13)
      at /opt/project/node_modules/telegraf/composer.js:337:14
      at execute (/opt/project/node_modules/telegraf/composer.js:352:13)
      at /opt/project/node_modules/telegraf/composer.js:337:14
```
## Failure Information (for bugs)

I'm not sure if it is a bug or I did something wrong. I just want to understand the reason or help to fix it.
For some time Telegraf works with no violation. After an undetected time span it fails.

### Steps to Reproduce

Unknown so far. Trying to update to the latest version and observe.

### Failure Logs

See above.

**Comment by sagadav** [comment]
Do you use composer in your code? If yes, attach the code. If not, try using a VPN

**Comment by vrenda16** [comment]
I have the same issue, but in my code i don't use composer. Some solution?

**Comment by KnorpelSenf** [comment]
Does this still happen when building from source? This might have been fixed by #1159

**Comment by wojpawlik** [comment]
If the issue persists, try reproducing it with low `--stack-trace-limit`.
